src/zui/ZUIOrganizeSidebar.tsx

```diff
@@ -1,219 +1,222 @@
-                size="large"
-                style={{ marginBottom: '2rem' }}
-              >
-                <ZUILogo beta={true} htmlColor="#ED1C55" size={40} />
-              </IconButton>
-            </NextLink>
-          </ListItem>
-          <ListItem disableGutters>
-            <NextLink href="/organize/" passHref>
-              <IconButton
-                aria-label="Home"
-                className={classes.roundButton}
-                color={key === '' ? 'primary' : 'secondary'}
-                data-test="home-button"
-                size="large"
-              >
-                <Home />
-              </IconButton>
-            </NextLink>
-          </ListItem>
-          <ListItem disableGutters>
-            <NextLink href={`/organize/${orgId}/people`} passHref>
-              <IconButton
-                aria-label="People"
-                className={classes.roundButton}
-                color={key.startsWith('/people') ? 'primary' : 'secondary'}
-                data-test="people-button"
-                size="large"
-              >
-                <People />
-              </IconButton>
-            </NextLink>
-          </ListItem>
-          <ListItem disableGutters>
-            <NextLink href={`/organize/${orgId}/journeys`} passHref>
-              <IconButton
-                aria-label="Journeys"
-                className={classes.roundButton}
-                color={key.startsWith('/journeys') ? 'primary' : 'secondary'}
-                data-test="people-button"
-                size="large"
-              >
-                <Explore />
-              </IconButton>
-            </NextLink>
-          </ListItem>
-          <ListItem disableGutters>
-            <NextLink href={`/organize/${orgId}/areas`} passHref>
-              <IconButton
-                aria-label="Areas"
-                className={classes.roundButton}
-                color={key.startsWith('/areas') ? 'primary' : 'secondary'}
-                data-test="area-button"
-                size="large"
-              >
-                <Map />
-              </IconButton>
-            </NextLink>
-          </ListItem>
-          <ListItem disableGutters>
-            <NextLink href={`/organize/${orgId}/projects`} passHref>
-              <IconButton
-                aria-label="Projects"
-                className={classes.roundButton}
-                color={key.startsWith('/projects') ? 'primary' : 'secondary'}
-                data-test="calendar-button"
-                size="large"
+  const [open, setOpen] = useState(false);
+  const toggleDrawer = () => {
+    setOpen(!open);
+    setExpanded(!expanded);
+  };
+
+  const model = useModel((env) => new OrganizationsDataModel(env));
+
+  const menuItemsMap = [
+    { icon: <Map />, name: 'areas' },
+    { icon: <Home />, name: 'home' },
+    { icon: <Explore />, name: 'journeys' },
+    { icon: <People />, name: 'people' },
+    { icon: <Event />, name: 'projects' },
```

Comment:
This array defines the order of the items in the menu, and here you've ordered them alphabetically, but I don't believe that's the order they should be in, according to the design mockups, so they need to be reordered here.

Reply:
Lint is complaining then because it likes to be alphabetically ordered.. can I disable the rule here? or do you know another approach?

Reply:
I don't think that's true. The linter should not care about the order of items in an array. Try reordering them, and I think you'll find that lint does not complain. Maybe it was complaining about `icon` and `name` being in the wrong order at some point? But that's unrelated to the order of the objects in the array.

Reply:
Correct! Sorry my mistake!